### PR TITLE
Add robo to minify CSS and JS

### PR DESCRIPTION
**Pull request by trasher**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "require": {
```
Comment (orthagh) [inline composer.json]: require-dev ?

Comment (trasher) [inline composer.json]: Indeed :)

**Comment by trasher** [comment]
Thinking about that, I wonder if we should not provide another robo file, say `RoboPluginFile.php` that would provide global tasks for all plugins; so each plugin can implement its own RoboFile and can update the global one easily.

**Comment by orthagh** [comment]
yes, i think its a good idea

**Comment by trasher** [comment]
OK, will do. Thanks!

File: RoboFilePlugin.php [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+   {
+      $css_dir = __DIR__ . '/css';
+      if (is_dir($css_dir)) {
+         foreach(glob("$css_dir/*.css") as $css_file) {
+            $this->taskMinify($css_file)
+               ->to("$css_file.min")
+               ->type('css')
+               ->run();
+         }
+      }
```
Comment (orthagh) [inline RoboFilePlugin.php]: I did some tests on my side, your sub functions should return $this to be chained like this in minify function:
```php
$this->minifyCSS()
     ->minifyJS();
```

Comment (trasher) [inline RoboFilePlugin.php]: I'm not sure this is a must have for this, but I can change that.

**Comment by trasher** [comment]
Oh, I've just realized that I rename files adding `.min` at the end, that is not correct, the `.min` must be added before the real file extension.

**Pull request merged**